docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2952,11 +2954,11 @@ Create a volume
       "Name": "tardis",
       "Driver": "local",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
```

Comment:
Also; perhaps we should have a `**JSON fields**:` section, describing the fields (we currently only have that for the *request*, not for the *response*, so fields that cannot be set in the request, but will be returned by the response currently don't have a description